tools/klee/main.cpp

```diff
@@ -291,7 +291,12 @@ KleeHandler::KleeHandler(int argc, char **argv)
     for (; i <= INT_MAX; ++i) {
       SmallString<128> d(directory);
       llvm::sys::path::append(d, "klee-out-");
-      raw_svector_ostream ds(d); ds << i; ds.flush();
+      raw_svector_ostream ds(d);
+      ds << i;
+// SmallString is always up-to-date, no need to flush. See Support/raw_ostream.h
```

Comment:
Done in https://github.com/llvm-mirror/llvm/commit/d4177b27058cfed2320f3ea1bf7ac2552f5a3e71